exercises/tuples-lists-dictionaries/challenge_075.py

```diff
@@ -0,0 +1,19 @@
+# TUPLES, LISTS AND DICTIONARIES
+
+# Challenge 075
+"""
+Create a list of four three-digit numbers. Display the list to the user, 
+showing each item from list on a separate line. Ask the user to enter
+a three-digit number. If the number they have typed in matches one in the list, 
+display the position of that number in the list, otherwise display the message
+"That is not in the list".
+"""
+
+numbers = [123, 345, 234, 765]
```

Comment:
This looks good. As a separate exercise it is worth trying to generate the four numbers randomly so they are different each time the script runs.

Reply:
Added the random numbers generator